readme.md

```diff
@@ -403,6 +404,11 @@ Very interesting exploration of what happens when aliens arrive on earth, after
 >
 > Jodahs is supposed to be just another hybrid of human and Oankali, but as he begins his transformation to adulthood he finds himself becoming ooloi—the first ever born to a human mother. As his body changes, Jodahs develops the ability to shapeshift, manipulate matter, and cure or create disease at will. If this frightened young man is able to master his new identity, Jodahs could prove the savior of what’s left of mankind. Or, if he is not careful, he could become a plague that will destroy this new race once and for all.
 
+#### [The Voyage of the Space Beagle](https://www.goodreads.com/book/show/1981484.The_Voyage_of_the_Space_Beagle) (1950) *by [A.E. Van Vogt](https://www.goodreads.com/author/show/1293688.A_E_van_Vogt)* [3.9]
+
+This novel reminds me of a season of _Star Trek_ episdoes, if Roddenberry's final frontier been Machiavellian. One section is very reminiscent of the _Alien_ plot. - [@neontapir](https://github.com/neontapir)
```

Comment:
Typo

---

> This novel reminds me of a season of _Star Trek_ episdoes, if Roddenberry's final frontier been Machiavellian.

Maybe you could expand on this. It would be useful to know what you mean even if one haven't watched that exact episode.